src/xcode/ENA/ENA/Source/Scenes/FriendsInvite/InviteFriendsViewController.swift

```diff
@@ -45,6 +45,11 @@ final class InviteFriendsViewController: UIViewController, UIActivityItemSource
 		inviteButton.setTitle(AppStrings.InviteFriends.submit, for: .normal)
 	}
 
+	override func viewWillAppear(_ animated: Bool) {
```

Comment:
Same question here. See above.

Reply:
See answer on DataDonationViewController.
InviteFriendsViewController does not inherit from UITableViewController, and I saw no adverse effect when I moved these lines.